packages/grid/x-grid-modules/src/components/styled-wrappers/getStyles.ts

```diff
@@ -0,0 +1,212 @@
+    borderRadius: theme.shape.borderRadius,
+    outline: 'none',
+    display: 'flex',
+    flex: 1,
+    flexDirection: 'column',
+
+    '& *': {
+      boxSizing: 'border-box',
+    },
+
+    '& .main-grid-container': {
+      position: 'relative',
+      flexGrow: 1,
+      flexDirection: 'column',
+      display: 'flex',
+    },
+    '& .footer': {
+      display: 'flex',
+      justifyContent: 'space-between',
+      flexDirection: 'row',
+      padding: theme.spacing(0, 2),
+    },
+    '& .row-count, & .selected-row-count': {
+      display: 'flex',
+      alignItems: 'center',
+      fontSize: '0.875rem',
+      fontWeight: theme.typography.fontWeightMedium,
+      lineHeight: 1.43,
+      minHeight: '48px',
+    },
+    '@media (max-width: 650px)': {
+      '&  .row-count, &  .selected-row-count': {
+        display: 'none',
+      },
+    },
+    '&  .MuiDataGrid-cell:focus, &  .MuiDataGrid-col-cell:focus': {
+      outline: 'dotted',
+      outlineWidth: '1px',
+      outlineOffset: '-2px',
+    },
+    '&  .overlay': {
+      display: 'flex',
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      right: 0,
+      bottom: '15px',
+      alignSelf: 'center',
+      alignItems: 'center',
+      zIndex: 10,
+    },
+    '&  .overlay .content': {
+      flex: 1,
+      display: 'flex',
+      justifyContent: 'center',
+    },
+
+    '&  .columns-container': {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      right: 0,
+      overflowX: 'hidden',
+      overflowY: 'hidden',
+      display: 'flex',
+      flexDirection: 'column',
+      borderBottom: `1px solid ${borderColor}`,
+      zIndex: 100,
+      backgroundColor: '#f9f9f9',
+      color: theme.palette.text.primary,
+      fontWeight: theme.typography.fontWeightBold,
+      fontSize: theme.typography.fontSize,
+    },
+    '&  .columns-container .MuiDataGrid-col-cell-wrapper': {
+      display: 'flex',
+      width: '100%',
+      alignItems: 'center',
+    },
+    '&  .MuiDataGrid-col-cell': {
+      position: 'relative',
+      display: 'flex',
+      padding: '0 16px',
+    },
+    '&  .MuiDataGrid-col-cell.sortable': {
+      cursor: 'pointer',
+    },
+
+    '&  .MuiDataGrid-col-cell.center': {
+      justifyContent: 'center',
+    },
+
+    '&  .MuiDataGrid-col-cell.right': {
+      justifyContent: 'flex-end',
+    },
+
+    '& .MuiDataGrid-col-cell .title': {
+      textOverflow: 'ellipsis',
+      overflow: 'hidden',
+      whiteSpace: 'nowrap',
+    },
+    '& .MuiDataGrid-col-cell .column-separator': {
+      position: 'absolute',
+      right: '-12px',
+      zIndex: 100,
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+    },
+    '& .MuiDataGrid-col-cell .column-separator .icon.separator': {
+      color: '#bdc3c7',
+    },
+    '& .MuiDataGrid-col-cell .column-separator:hover .separator.resizable': {
+      cursor: 'col-resize',
+      color: 'inherit',
+    },
+    '&  .MuiDataGrid-col-cell *': {
```

Comment:
Why do we need a wildcard selector?

```suggestion
    '&  .MuiDataGrid-col-cell': {
```